src/app/Http/Middleware/AuthenticateSession.php

```diff
@@ -0,0 +1,112 @@
+<?php
+
+namespace Backpack\CRUD\app\Http\Middleware;
+
+use Closure;
+use Illuminate\Auth\AuthenticationException;
+use Illuminate\Contracts\Auth\Factory as AuthFactory;
+use Illuminate\Contracts\Session\Middleware\AuthenticatesSessions;
+
+class AuthenticateSession implements AuthenticatesSessions
+{
+    /**
+     * The authentication factory implementation.
+     *
+     * @var \Illuminate\Contracts\Auth\Factory
+     */
+    protected $auth;
+
+    /**
+     * Create a new middleware instance.
+     *
+     * @param  \Illuminate\Contracts\Auth\Factory  $auth
+     * @return void
+     */
+    public function __construct(AuthFactory $auth)
+    {
+        $this->auth = $auth;
+        $this->user = backpack_user();
```

Comment:
Just a small not here, in PHP 8.2, this will throw an error because the `user` property is not defined prior to it's definition. 

If we want to think ahead we can probably add it already, I've added a comment for you to push if you think we should.